RollbarCrashReport.podspec

```diff
@@ -1,30 +1,34 @@
 Pod::Spec.new do |s|
-  s.name         = "RollbarCrashReport"
-  s.version      = "3.0.0"
-  s.summary      = "Application or client side SDK for interacting with the Rollbar API Server."
-  s.description  = <<-DESC
-                    Find, fix, and resolve errors with Rollbar.
-                    Easily send error data using Rollbar API.
-                    Analyze, de-dupe, send alerts, and prepare the data for further analysis.
-                    Search, sort, and prioritize via the Rollbar dashboard.
-                 DESC
+    s.name         = "RollbarCrashReport"
+    s.version      = "3.0.0"
+    s.summary      = "Application or client side SDK for interacting with the Rollbar API Server."
+    s.description  = <<-DESC
+                      Find, fix, and resolve errors with Rollbar.
+                      Easily send error data using Rollbar API.
+                      Analyze, de-dupe, send alerts, and prepare the data for further analysis.
+                      Search, sort, and prioritize via the Rollbar dashboard.
+                  DESC
 
-  s.homepage     = "https://rollbar.com"
-  s.resource     = "rollbar-logo.png"
-  s.license      = { :type => "MIT", :file => "LICENSE" }
-  s.authors      = { "Rollbar" => "support@rollbar.com" }
-  s.source       = { :git => "https://github.com/rollbar/rollbar-apple.git",
-                     :tag => s.version.to_s }
-  s.documentation_url = "https://docs.rollbar.com/docs/apple"
-  s.social_media_url  = "http://twitter.com/rollbar"
+    s.homepage     = "https://rollbar.com"
+    s.resource     = "rollbar-logo.png"
+    s.license      = { :type => "MIT", :file => "LICENSE" }
+    s.authors      = { "Rollbar" => "support@rollbar.com" }
+    s.source       = { :git => "https://github.com/rollbar/rollbar-apple.git",
+                       #:tag => s.version.to_s }
```

Comment:
Is it important to keep the commented-out line?

Reply:
No, this is a mistake! <3

Reply:
Branch was being used to lint the spec since it has to download code from git. And I forgot to revert this back to the version tag.